projects/fsastorefrontlib/src/assets/translations/en/b2b.en.ts

```diff
@@ -12,10 +12,86 @@ export const organization = {
     productAssignment: 'Product Assignment',
   },
   orgUnit: {
+    header: 'All organization ({{count}})',
+    unit: 'Organization',
+    parentUnit: 'Parent Organization',
+    details: {
+      title: 'Organization Details',
+      subtitle: 'Organization: {{ item.name }}',
+    },
+    edit: {
+      title: 'Edit Organization',
+      subtitle: 'Organization: {{ item.name }}',
+    },
+    create: {
+      title: 'Create Organization',
+      subtitle: '',
+    },
+    messages: {
+      deactivate: 'Are you sure you want to disable this organization?',
+      confirmEnabled: 'Organization {{item.name}} enabled successfully',
+      confirmDisabled: 'Organization {{item.name}} disabled successfully',
+      update: 'Organization {{ item.name }} updated successfully',
+      create: 'Organization {{ item.name }} created successfully',
+    },
+    links: {
+      units: 'Child Organization',
+    },
+    children: {
+      create: {
+        title: 'Create child Organization',
+        subtitle: '',
+      },
+      messages: {
+        create: 'Organization {{ item.name }} created successfully',
+      },
+    },
+    form: {
+      parentOrgUnit: 'Parent business organization',
+      create: 'Create Organization',
+    },
+    assignApprovers: {
+      instructions: {
+        check: `To assign an approver to this organization, select the user's check box.`,
+      },
+    },
     breadcrumbs: {
+      list: 'All organization',
+      children: 'Child organization',
       productAssignment: 'Product Assignment',
     },
   },
+  orgUnitChildren: {
+    title: 'Child organization',
+    subtitle: 'Organization: {{item.name}}',
+  },
+  orgUnitApprovers: {
+    subtitle: 'Organization: {{item.name}}',
+  },
+  orgUnitAssignedApprovers: {
+    subtitle: 'Organization: {{item.name}}',
+  },
+  orgUnitAssignedUsers: {
+    subtitle: 'Organization: {{item.name}}',
+  },
+  orgUnitUsers: {
+    subtitle: 'Organization: {{item.name}}',
+  },
+  orgUnitCostCenters: {
+    subtitle: 'Organization: {{item.name}}',
+  },
+  orgUnitAddress: {
```

Comment:
Do we really use these translations? Also for cost centers above.

Reply:
Actually I am not sure and don't know how to check it.

Reply:
For cost centers I'm 100% sure that we don't use it but for other things you can just quickly check it over storefront. If there is no possibility to add some address and stuff you can delete these translations.